packages/block-library/src/cover/test/edit.native.js

```diff
@@ -54,12 +54,12 @@ const COVER_BLOCK_SOLID_COLOR_HTML = `<!-- wp:cover {"overlayColor":"cyan-bluish
 <!-- /wp:paragraph --></div></div>
 <!-- /wp:cover -->`;
 const COVER_BLOCK_IMAGE_HTML = `<!-- wp:cover {"url":"https://cldup.com/cXyG__fTLN.jpg","id":10710,"dimRatio":50,"overlayColor":"foreground","isDark":false} -->
-<div class="wp-block-cover is-light"><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim"></span><img class="wp-block-cover__image-background wp-image-10710" alt="" src="https://cldup.com/cXyG__fTLN.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"large"} -->
+<div class="wp-block-cover is-light"><img class="wp-block-cover__image-background wp-image-10710" alt="" src="https://cldup.com/cXyG__fTLN.jpg" data-object-fit="cover"/><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"large"} -->
```

Comment:
I think these need updating further.  This change brings the HTML markup only up-to-date. It's missing the `isUserOverlayColor` true attribute which might be causing the removal of that attribute in the snaps that have also been updated.

Reply:
Thank you for your careful reviews and your patience, @aaronrobertshaw!

I updated these commits to fix this: https://github.com/WordPress/gutenberg/pull/66249/commits/eaac4cd6505712044cee563b188c78e028840da7 https://github.com/WordPress/gutenberg/pull/66249/commits/9ae33452a31490868a47a6f204adfc257cd5809e

Honestly, I didn't investigate deeper the reasons to understand better, but I suspect it's because we added this attribute to the deprecation now (`v14BlockAttributes`). If you are not confident that it's really needed and we still need some investigation, let me know and I can investigate this more another time. 😉